whipper/test/test_result_logger.py

```diff
@@ -163,7 +163,8 @@ def testLogger(self):
                 Dumper=ruamel.yaml.RoundTripDumper
             )
         )
+        log_body = "\n".join(actualLines[:-1]).encode()
         self.assertEqual(
             parsedLog['SHA-256 hash'],
-            hashlib.sha256("\n".join(actualLines[:-1])).hexdigest().upper()
+            hashlib.sha256(log_body).hexdigest().upper()
```

Comment:
Why not .encode() inline?

Reply:
Putting that inline would make the line lenght exceed PEP8 limit and I think it's clearer the current way.

Reply:
OK, actually did not know about PEP8 line limit. Both of your point make sense.